src/app/shared/object-list/search-result-list-element/item-search-result/item-types/item/item-search-result-list-element.component.html

```diff
@@ -18,7 +18,7 @@
           <div role="button" class="btn-link" (click)="showSettings()">{{ "third-party-metrics-cookies.consent-settings" | translate }}</div>
         </small>
         <div class="d-flex">
-          <ds-themed-badges *ngIf="showLabel" [object]="dso" [context]="context" [showAccessStatus]="true"></ds-themed-badges>
+          <ds-themed-badges *ngIf="showLabel" [object]="dso" [context]="context" [showAccessStatus]="false"></ds-themed-badges>
```

Comment:
not really necessary because we never have 'accessStatus' information, so this badge never appears